jest.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
       files: ['*.test.[tj]s?(x)'],
       rules: {
         'testing-library/no-node-access': 'off', // TODO this rule was enabled for new testing-library
+        'jest/expect-expect': ['warn', { assertFunctionNames: ['expect', '*expect*'] }],
```

Comment:
Note from Jordi:
relaxes jest/expect-expect a bit by allow expect to be used instead of only expect() - useful if expectations are in a helper function instead of inside the test/it callback.
Makes it so that this warning goes away because Expect is in the function distantRelativeFlowExpectations.

Without updated rule:
![image](https://user-images.githubusercontent.com/3885959/215877142-73a4173d-5e2c-4c68-bf89-abdd622502b2.png)
of course you could just disable it by a case by case example though with something like: // eslint-disable-next-line jest/expect-expect -- assertions are in distantRelativeFlow

With updated rule

![image](https://user-images.githubusercontent.com/3885959/215877206-49e472ec-c817-480d-827a-55cef7ec89c9.png)


![image](https://user-images.githubusercontent.com/3885959/215877220-e237b571-d5a4-4b1e-8452-40def016760c.png)